OnlinerTests/Tests/BaseTest.cs

```diff
@@ -0,0 +1,29 @@
+﻿using NUnit.Framework;
+using OnlinerTests.PageObjects.Basic;
+using OpenQA.Selenium;
+
+namespace OnlinerTests.Tests
+{
+    class BaseTest
+    {
+        protected object[] data;
+
+        [SetUp]
+        public void AhSHereWeGoAgain()
+        {
+            WebDriverProvider.Start();
+        }
+
+        [TearDown]
+        public void YouHadToFollowTheDamnTrainCJ()
```

Comment:
1. give the method an appropriate name
2. it would be better to quit a driver before each suite rather than test
3. add a screenshot helper and write here a method that would make screenshot of the screen each time when a test fails

Reply:
I'll make screenshot helper;